python/src/KernelSmoothing_doc.i.in

```diff
@@ -22,28 +22,67 @@ dimension, then the unit weight of each point is linearly affected to the vertic
 of the bin containing the point. See [wand1994]_ for the details.
 
 The boundary correction is available only in one dimension, and it is done using
-the mirroring technique. See the notes of the *setBoundingOption* method for the
+the mirroring technique. See the notes of the :meth:`setBoundingOption` method for the
 details.
 
 When applied to multivariate samples, the kernel is the kernel product of the
-univariate distribution specified in the constructor. 
+univariate distribution specified in the constructor.
 
 Examples
 --------
 Fit a distribution on data thanks to the kernel smoothing technique:
 
 >>> import openturns as ot
 >>> sample = ot.Gamma(6.0, 1.0).getSample(100)
->>> kernel = ot.KernelSmoothing()
->>> fittedDist = kernel.build(sample)
+>>> ks = ot.KernelSmoothing()
+>>> fittedDist = ks.build(sample)
+>>> print(fittedDist.getClassName())
+Distribution
```

Comment:
not sure how this is useful

Reply:
I assume the idea was to showcase the various build/buildAsXxx methods. The main difference for users is the type of the output.

Reply:
then it needs a comment to explain that

Reply:
Please have a look at my latest commit. Do you find this clarification satisfactory?